src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -746,6 +746,14 @@ export const needsToAnswerUnemployability = formData =>
   needsToEnterUnemployability(formData) &&
   _.get('view:unemployabilityUploadChoice', formData, '') === 'answerQuestions';
 
+export const doctorsCare = formData =>
+  needsToAnswerUnemployability(formData) &&
+  _.get('view:medicalCareType.view:doctorsCare', formData, false);
+
+export const hospitalCare = formData =>
```

Comment:
better names, something like `hasDoctorsCare` to indicate it's returning a boolean

Reply:
Also unit tests

Reply:
Can check out my unit tests? pushed a new commit. @zurbergram

Reply:
though the format seems to be `needs`